deployment/README.md

```diff
@@ -1 +1,41 @@
-# Deployment
+# Amazon Web Services Deployment
+
+Amazon Web Services deployment is driven by [Terraform](https://terraform.io/) and the [AWS Command Line Interface (CLI)](http://aws.amazon.com/cli/).
+
+## Table of Contents
+
+* [AWS Credentials](#aws-credentials)
+* [Terraform](#terraform)
+
+## AWS Credentials
+
+Using the AWS CLI, create an AWS profile named `raster-foundry`:
+
+```bash
+$ vagrant ssh
+vagrant@vagrant-ubuntu-trusty-64:~$ aws --profile raster-foundry configure
+AWS Access Key ID [****************F2DQ]:
+AWS Secret Access Key [****************TLJ/]:
+Default region name [us-east-1]: us-east-1
+Default output format [None]:
+```
+
+You will be prompted to enter your AWS credentials, along with a default region. These credentials will be used to authenticate calls to the AWS API when using Terraform and the AWS CLI.
+
+## Terraform
+
+Next, use the Terraform wrapper script (`terraform.sh`) to lookup the remote state of the infrastructure and assemble a plan for work to be done:
+
+```bash
+vagrant@vagrant-ubuntu-trusty-64:~$ export RF_SETTINGS_BUCKET="raster-foundry.staging.config.us-east-1"
```

Comment:
I'm wondering if we should start shying away from using `.` as a separator for buckets so that we can start taking advantage of s3 acceleration. That's not relevant for the configs, but trying to think ahead for how we will name other buckets.